Dockerfile

```diff
@@ -468,15 +468,15 @@ ARG CGO_ENABLED=1
 ARG GOARCH_aarch64="arm64"
 ARG GOARCH_x86_64="amd64"
 ARG GOARCH_ARCH="GOARCH_${ARCH}"
-ARG CFLAGS="-O2 -g -pipe -Wformat -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-clash-protection"
+ARG CFLAGS="-O2 -g -pipe -Wall -Werror=format-security -Wp,-D_FORTIFY_SOURCE=2 -Wp,-D_GLIBCXX_ASSERTIONS -fexceptions -fstack-clash-protection"
 ARG CXXFLAGS="${CFLAGS}"
 ARG LDFLAGS="-Wl,-z,relro -Wl,-z,now"
 ARG CGO_CFLAGS="${CFLAGS}"
 ARG CGO_CXXFLAGS="${CXXFLAGS}"
 ARG CGO_LDFLAGS="${LDFLAGS}"
 
 WORKDIR /home/builder/sdk-go/src
-RUN ./all.bash
+RUN ./make.bash
```

Comment:
Do any of the tests fail? Can we keep `./all.bash`?

Reply:
I suspect they would pass, but I'd rather add that back in as part of https://github.com/bottlerocket-os/bottlerocket-sdk/pull/148 or a fast-follow.